src/components/Card/Card.js

```diff
@@ -8,22 +8,34 @@ import {
   BsArchive,
   BsArchiveFill,
 } from "react-icons/bs";
+import { MdClose } from "react-icons/md";
 import ColorPalette from "./Colors/Colors";
+import { useTrashContext } from "../../context";
+import { useLocation } from "react-router-dom";
 
+export const Card = (note) => {
+  let {
+    id,
+    title,
+    archived,
+    body,
+    createdAt,
+    updatedAt,
+    pinned,
+    trashed,
+    tags,
+    color,
+  } = note;
+
+  const { deleteNote, restoreNote, deleteFromTrash } = useTrashContext();
+
+  const location = useLocation();
+  console.log(location.pathname);
```

Comment:
Remove the unused console logs.

Reply:
Yup, in my future PRs I have removed them.